src/SignalR/common/Http.Connections/src/Internal/HttpConnectionContext.cs

```diff
@@ -323,7 +323,7 @@ private async Task WaitOnTasks(Task applicationTask, Task transportTask, bool cl
             }
         }
 
-        public bool TryActivatePersistentConnection(
+        internal bool TryActivatePersistentConnection(
```

Comment:
Why only this method?

Also, this class is still pubternal.

Reply:
Why is this class pubternal still?

Reply:
> Why only this method?

Because `IHttpTransport` is  internal now. 

>Also, this class is still pubternal.

Yeah, I updated the issue with the new namespace that was missed and I'm updating those types now.

Reply:
From the issue.

> Edit More Pubternal Types
> Microsoft.AspNetCore.Http.Connections.Internal.

> HttpConnectionContext
> HttpConnectionDispatcher
> HttpConnectionManager
> HttpConnectionStatus

Looking at the [ref assembly](https://github.com/aspnet/AspNetCore/blob/017782ccc576c92164bd31b49db7df4f517fd21a/src/SignalR/common/Http.Connections/ref/Microsoft.AspNetCore.Http.Connections.netcoreapp3.0.cs) it looks like we should add AwaitableThreadPool to the list.

Reply:
That’s public?

Reply:
I think it was pubternal so it could be called from Http.Connections and Client.Core, but we're much better off just copying it.

Reply:
Shared codeZ

Reply:
>  like we should add AwaitableThreadPool to the list

Yeah, I made it internal but forgot to add it to the issue.